src/mechaphlowers/api/frames.py

```diff
@@ -201,20 +227,22 @@ def add_cable(self, cable: CableArray):
 			ValueError: if cable has not the same length as the section
 		"""
 
-		if not isinstance(cable, CableArray):
-			raise TypeError("cable has to be a CableArray object")
+		property_map = {
+			CableArray: "cable",
+			SectionArray: "section",
+			WeatherArray: "weather",
+		}
+
+		if not isinstance(var, type_var):
+			raise TypeError(f"{type_var} has to be a CableArray object")
```

Comment:
Update error message:
```suggestion
			raise TypeError(f"var has to be a {type_var} object")
```
Also, do we use 
`{type_var}`, which displays `<class 'mechaphlowers.entities.arrays.CableArray'>` or
`{type_var}.__name__` which displas `'CableArray'`

Reply:
Better to use {type_var}.__name__ which displas 'CableArray' indeed. Thank you !